test/Test/TZBTC.hs

```diff
@@ -506,9 +506,10 @@ test_approvableLedgerV1 = testGroup "TZBTC contract approvable ledger tests"
       :: (MonadCleveland caps m)
       => ImplicitAddress -> AL.AlSettings -> m (ContractHandle (Parameter TZBTCv1) (Storage TZBTCv1) ())
     alOriginate = originateManagedLedger $ \op  -> do
-      let owner = case ML.opAdmin op of
+      let owner' = case ML.opAdmin op of
             MkAddress x@ImplicitAddress{} -> x
             _ -> error "Unexpected address type in alOriginate"
+      owner <- refillable $ pure owner' -- force owner to be refillable
```

Comment:
Why not
```suggestion
      owner <- case ML.opAdmin op of
            MkAddress x@ImplicitAddress{} -> refillable $ pure x -- force owner to be refillable
            _ -> error "Unexpected address type in alOriginate"
```

Reply:
I've opted to move `refillable . pure` outside `case` to make `refillable` more obvious.